Agreements/Internal/TCSW_S.txt

```diff
@@ -11,6 +11,8 @@ COPX:EGLK:*:SAM:*:*:London TC SW:London AC Worthing:15000:*:^SAM
 COPX:EGTF:*:SAM:*:*:London TC SW:London AC Worthing:15000:*:^SAM
 COPX:EGHL:*:SAM:*:*:London TC SW:London AC Worthing:15000:*:^SAM
 
+COPX:EGLL:*:EXARO:*:*:London TC SW:London AC Worthing:15000:*:^SAM
```

Comment:
I suppose ^SAM makes sense in terms of situational awareness, but does seem a bit odd to say something is climbing via something it is not. @cpawley could this be a reason to just have these agreements as the arrow, without mentioning a point?

Reply:
In theory the point only exists as a situational aid (though this is the idea of intention codes, right?) as the agreement is just climbing with no level by point. The climbing arrow is surely "more correct" in this scenario?

Reply:
^EXARO?

Reply:
Changed to ^EXARO